src/main/java/com/andersenlab/lesson2/abstraction/IdAbler.java

```diff
@@ -0,0 +1,13 @@
+package com.andersenlab.lesson2.abstraction;
+
+public abstract class IdAbler {
```

Comment:
It is not the best name for abstract class with some essential attributes, what if you will add one more field here?